### ISSUE TITLE
the OFL-1.1 license violates 17 USC § 105

### ISSUE THREAD

**Issue by mbutterick** [uswds/public-sans#30]
[17 USC § 105](https://www.law.cornell.edu/uscode/text/17/105) says that any work of a US government employee is not subject to copyright. The OFL-1.1 license, by contrast, depends on the assertion of copyright for its effect. Therefore, there is no way for the work of US gov’t employees to be released under OFL-1.1. (I take no position on whether the terms of OFL-1.1 have been correctly applied here, though I’m skeptical.)

**Comment by thisisdano** [comment]
Hm — where do you think we could go from here? Is there a more appropriate license that we could use, perhaps in coordination and discussion with the original creator of the face?

cc @impallari

**Comment by mbutterick** [comment]
I think `public-sans` is already in the public domain because it cannot be otherwise. Under that view, Mr. Impallari’s copyright has been invalidated by inverse condemnation. IIRC Mr. Impallari is a citizen of [Argentina](https://github.com/impallari). This means that this project has also possibly violated the Buy American Act, [41 USC § 8302](https://www.law.cornell.edu/uscode/text/41/8302).

**Comment by pbhj** [comment]
The exception in 8302 (a) 2 C would appear to make it perfectly acceptable against the "Buy American Act", assuming there was no charge?

Also, Argentina is in America, why isn't it called "Buy from USA Act"?

You mention the nationality of the creator, but that Act talks about the location of manufacture.

**Comment by mbutterick** [comment]
Furthermore, by calling this font `public-sans` (and not, say, `copyrighted-by-someone-else-sans`)  the US gov’t has already made an explicit representation that it is meaningfully “public”.

**Comment by pbhj** [comment]
IIRC (I'm not from USA) the gov can produce things that are "public domain" in USA, but copyright elsewhere.

Of course then anyone in USA could republish a "public domain" work as truly public domain, circumventing the gov's restricted right in other jurisdictions.

The only sensible license is "public domain", surely?

**Comment by evbarnett** [comment]
I don't believe Public Sans is in the public domain, since it is based off of Libre Franklin. It must adhere to Libre Franklin's license, and I don't think this law applies anyway because Libre Franklin was not the work of a government employee.

**Comment by sixty4k** [comment]
`Copyright protection under this title is not available for any work of the United States Government, but the United States Government is not precluded from receiving and holding copyrights transferred to it by assignment, bequest, or otherwise.`

Worth considering, here and in #31, is that this is a derivative work of Libre Franklin, which is licensed by the OFL-1.1.  That license allows for modification and redistribution but not relicensing.  I think that falls comfortably under `holding copyrights transferred to it by assignment, bequest, or otherwise.`

**Comment by tssva** [comment]
> I think that falls comfortably under holding copyrights transferred to it by assignment, bequest, or otherwise.

This is incorrect . The OFL-1.1 just grants certain rights of use to the work but in no way transfers the copyright for the work to the government. The copyright is fully maintained by the original creator.

**Comment by sixty4k** [comment]
@tssva you're right.  I think that _might_ negate this.  As a derivative work, the copyright is not in question.

**Comment by david-a-wheeler** [comment]
Additional work done by a federal government employee as part of that employee's official duties is not subject to copyright in the United States.  However, anything from the original work retains its copyrights, and this doesn't apply outside the United States.  If they are reporting the license as a whole, then OFL-1.1 is correct, and as far as I know there is no violation of 17 USC 105.

Disclaimer: I am not a lawyer (IANAL).  But I suspect a real lawyer would say the same thing.

**Comment by jackjennings** [comment]
FWIW @mbutterick _is_ a lawyer and type designer…

**Comment by tssva** [comment]
> However, anything from the original work retains its copyrights, and this doesn't apply outside the United States. If they are reporting the license as a whole, then OFL-1.1 is correct, and as far as I know there is no violation of 17 USC 105.

The OFL-1.1 only allows distribution of derivative works if the derivative is itself licensed under the OFL-1.1, so if you take the view that public-sans is a combination of public domain licensed and OFL-1.1 licensed work than distribution of it is a violation of the original works OFL-1.1 license.

**Comment by mbutterick** [comment]
> where do you think we could go from here? Is there a more appropriate license that we could use, perhaps in coordination and discussion with the original creator of the face?

Though I am a lawyer, I obviously cannot advise the GSA on legal matters. Nothing I’ve said in this thread is legal advice. I am simply flagging a problem with the licensing.

**Comment by fredrikhemsen** [comment]
> > where do you think we could go from here? Is there a more appropriate license that we could use, perhaps in coordination and discussion with the original creator of the face?
> 
> Though I am a lawyer, I obviously cannot advise the GSA on legal matters. Nothing I’ve said in this thread is legal advice. I am simply flagging a problem with the licensing.

As far as I can tell, you are the only lawyer in this thread; so if anyone here were to know anything about federal law, it would be you.
In your newsletter you stated that “Public Sans is an inseparable mixture of copyrighted work […] and uncopyrightable work.” It seems that the creators of this typeface, somehow, managed to design themselves into a corner; a fact I find rather amusing.

**Comment by david-a-wheeler** [comment]
I wrote a journal article about open source software licensing issues:
"Publicly Releasing Open Source Software Developed for the U.S. Government"
https://www.csiac.org/journal-article/publicly-releasing-open-source-software-developed-for-the-u-s-government/
That paper specifically discusses modifying existing open source software, and I expect that fonts are not really different.

I also helped develop the Department of Defense policy on open source software.

There is nothing wrong with the government taking some existing open source software, modifying it, and then releasing that result from open source software. It is true that in general government employees cannot usually, as part of their duties, develop something that has copyright within the US (there are exceptions for NIST and the postal service).

But as far as I can tell that is not the claim being made here. The original work that this is derived from has a copyright license, and it allows derivatives under that same license. The modifications done by any US government employees would not be copyrighted within the US, but they can be released under that license outside the US, and the combined work as a whole would still be under the original license.

All legal questions are very fact-specific, so perhaps there is some issue. But I don't see any issues with the facts listed so far.

I am not a lawyer, but I have talked to a very large number of lawyers. If you're serious about a legal question, you really have to go to government lawyers, they are the only ones that can formally adjudicate government legal questions.

**Comment by david-a-wheeler** [comment]
The derivative is under the same license as the original, so I think it would be quite a bizarre argument that it does not satisfy the licensing requirement.

**Comment by fredrikhemsen** [comment]
> The derivative is under the same license as the original, so I think it would be quite a bizarre argument that it does not satisfy the licensing requirement.

I believe Matthew is not arguing that Public Sans doesn not satisfy the licensing agreement, but rather that the very fact that it _does_ is what makes the project in violation of 17 USC § 105.

**Comment by david-a-wheeler** [comment]
That is a highly unusual argument, and not one I've ever heard a government lawyer ever promulgate. The portions that the government developed are not copyrighted within the us, and that is all 105 requires. The work (when used as a whole) retains its original copyright license, because there are parts that have the original license... but that is not a situation that 105 prevents.

**Comment by mbutterick** [comment]
The GSA employs many fine lawyers who I’m sure are very excited to sort this out for the American people.

> The original work that this is derived from has a copyright license, and it allows derivatives under that same license.

Agreed. My point is that the OFL — and for that matter, any open-source license — presumes the licensor is holding a valid copyright for the thing that is to be licensed. Don’t take my word for it. See, e.g., [What is Copyleft?](https://www.gnu.org/licenses/copyleft.en.html) by the Free Software Foundation (emphasis mine):

“To copyleft a program, we first state that it is copyrighted … Copyleft is a way of using the copyright on the program. It doesn't mean abandoning the copyright; **in fact, doing so would make copyleft impossible.**”

In other words, copyright ownership is an indispensable prerequisite for the creation of an open-source license.

> the combined work as a whole would still be under the original license

Ah, but there’s the rub: to place the derivative work under the OFL, the GSA needs to be holding a copyright to its contributions, which it does not (and cannot) have. Because this is so, it’s not clear how it can ever fulfill the requirements of the OFL that covers Libre Franklin. 

You say in your article:

“If the government has unlimited rights, it has essentially the same rights as a copyright holder for purposes of releasing the software as OSS. Thus, it can release the software under any OSS license it chooses, including the GNU General Public License (GPL) and Lesser GPL (LGPL).”

Respectfully, I don’t see why this would be so. (If you have a case citation or agency interpretation, I would be happy to see it.) An open-source license is still a license: it seeks to put certain conditions & limitations on the use of the software. Consider that not all open-source licenses are the same. For instance, the OFL and GPL require that derivative works be released under comparable terms. This is not a limitation of, say, the MIT license. These restrictions are only possible by the assertion of copyright (and even the Free Software Foundation agrees). 

To put it another way: if you release software under an open-source license, you are representing that you hold a copyright on that software. This is why the licensing scheme for Public Sans runs afoul of 17 USC § 105.

BTW in 2014 the Smithsonian released a font family called [Cooper Hewitt](https://www.cooperhewitt.org/open-source-at-cooper-hewitt/cooper-hewitt-the-typeface-by-chester-jenkins/), also under the OFL. That was a different situation, however, because IIUC they hired an external contractor to modify the design and place it under the OFL. AFAIK there was no work by US gov’t employees baked into it, as there is here.

**Comment by david-a-wheeler** [comment]
> “If the government has unlimited rights, it has essentially the same rights as a copyright holder for purposes of releasing the software as OSS. Thus, it can release the software under any OSS license it chooses, including the GNU General Public License (GPL) and Lesser GPL (LGPL).”

That's a different case. Unlimited rights is often the result when a contractor does work for the government. That is covered in cases B, D, and E.

We are discussing case A, where a government employee has directly implemented software. You will notice that the discussion there is different, the point is that the government is allowed to release that software without copyright within the US, and is free to do all sorts of things outside the US (including using a traditional license).

**Comment by david-a-wheeler** [comment]
>  if you release software under an open-source license, you are representing that you hold a copyright on that software.

The US government can assert copyright outside the US on software it releases. So if you require a copyright assertion, there it is.

There is a special case: within the US. Within the US, no such copyright can be asserted for the government portions. However the work as a whole is still copyrighted and can only be used under that license. The entire work can only be released under the original license, and that has been done, so the license has been complied with.

Most users don't care about all these details, and just want to know under what condition they are allowed to use the entire work. The answer the same whether or not is within the US or outside the US.

**Comment by mbutterick** [comment]
> We are discussing case A, where a government employee has directly implemented software.

Fair enough. Let’s just consider the situation within the U.S. In that case you say the answer to the question “can government release [software] as OSS?” is “Effectively yes. The software is not subject to copyright protection in the U.S. … The government may … still release the software as OSS.” I don’t see where you address the question of how the government gets the copyright that is the necessary predicate for the OSS license within the U.S.. If there’s an answer to that, in your article or elsewhere, I’m curious to see it. (Or perhaps you disagree that a copyright is necessary predicate of an open-source license, contra the FSF.)

BTW, the difference between OFL and public domain is nontrivial. If Public Sans is public domain (as I contend it should be, and maybe already is) then it **vastly** enlarges what users can do with it. I think that would be a nice thing for people of the USA.

**Comment by david-a-wheeler** [comment]
> Or perhaps you disagree that a copyright is necessary predicate of an open-source license, contra the FSF.

Warning: extreme pedantry follows, because I don't see how to avoid it.

The FSF doesn't have much to say about open source software licenses, they emphasize free software and the free software definition. I presume you are referring to the OSI.

We must now carefully split the difference between open source software and open source software licenses. Software without copyright meets all of the parts of the open source definition and all of the parts of the free software definition. Therefore, software can be out of copyright, be open source software, and be free software in the free software definition sense, all at the same time. The OSI chooses to only evaluate certain licenses, and that is their right, but it does not change the fact that software without copyright still meets the definition.

All software has limited copyright, and eventually it will go out of copyright. When the copyright expires, does that mean it is no longer open source software? That makes no sense.  Eventually all software written today will not be open source software simply because their copyright expires if that were the case, even though such software meets the definitions of both the open source definition and the free software definition.

**Comment by david-a-wheeler** [comment]
By the way, the FSF specifically notes that source code in the public domain is free software per the free software definition:
https://www.gnu.org/philosophy/categories.en.html

So the FSF certainly accepts the notion that software released without a copyright is free software.

**Comment by wslack** [comment]
IANAL (and I'm writing this personally, to be clear), but I think this thread has gone down a rabbit hole that isn't helpful. (with much love to @david-a-wheeler for helpfully working through the points raised)

The work of US gov't employees is in the public domain. The original work that was transformed the US gov't employees is not.

> Public Sans is an inseparable mixture of copyrighted work […] and uncopyrightable work.

I agree with this. Put another way, I would say the project contains:

 - Open source works that are not in the public domain
 - Open source work by the U.S. government that is in the public domain

Perhaps the license information could be clearer to this point (I defer to @thisisdano) but I don't see a legal problem. If we say that work on a copyrighted work that is "inseparable" violates a legal requirement, that would forbid any transformations of copyrighted work by any US gov't employees, anytime.

**Comment by mbutterick** [comment]
_[Once again, I am not anyone’s lawyer and nothing I’m saying in this thread is legal advice.]_

> We must now carefully split the difference between open source software and open source software licenses … the FSF certainly accepts the notion that software released without a copyright is free software.

I see what you mean, but this effort shifts the goalposts. The OFL is a license; it requires that derived works be “[distributed entirely under](https://opensource.org/licenses/OFL-1.1)” the OFL. So even if public-domain works are morally open-source-ish (in the OSI sense) and free-software-ish (in the FSF sense), they still cannot be **licensed** in the way the OFL demands. You’re right that the OSI and FSF diverge on certain matters, but they agree ([here](https://opensource.org/node/878) and [here](https://www.gnu.org/philosophy/categories.en.html)) that a license necessarily derives from copyright. No copyright means no license.

>  IANAL … but I think this thread has gone down a rabbit hole … I don't see a legal problem.

Sir, you’ve perfectly described the legal problem: Public Sans is a combination of existing copyrighted work and new public-domain work. No one has argued that the public-domain work can be put under license within the US. So the GSA has left itself no way to “distribute[]” Public Sans ”entirely under” the OFL. This, in turn, raises a more-than-rabbit-sized question about what people can do with the fonts.

> If we say that work on a copyrighted work that is "inseparable" violates a legal requirement, that would forbid any transformations of copyrighted work by any US gov't employees, anytime.

It’s possibly knottier than that. Though the US gov’t has a special legal restriction (= it cannot create copyrighted works), it also has a special legal power (= it can remove private property to the public domain). So yes, I imagine US gov’t employees would need to step _very_ gingerly around “transformations of copyrighted work”, lest they accidentally nudge those works into the public domain. Given that the US gov’t has a safer way of handling copyrighted software — roughly, _use contractors_ — the GSA’s decision not to do so here may indeed have incurred other consequences. (Leaving aside whatever significance the GSA intended by calling it “Public Sans” and not, say, “Partially Copyrighted By Someone Else Sans”)

**Comment by mbutterick** [comment]
BTW — the OFL is a [license approved by the Open Source Initiative](https://opensource.org/licenses/OFL-1.1). The OSI, in turn, has [specifically warned](https://opensource.org/node/878) that “public domain is not open source” and that “**an open source user or developer cannot safely include public domain source code in a project**” (their emphasis) because it “leaves doubts” about the permitted uses.

**Comment by EssbieWGT** [comment]
@mbutterick just curious: 

- What cause of action is present and by whom if you're right about this OFL-1.1 issue? 
- What rights would the government have under OFL-1.1 that they do not possess if public-sans is in the public domain? 
- How can this issue have any effect on the copyrights of related fonts it's arguably derived from when the stated legislative intent of Section 105 is that "publication or other use by the Government of a private work would not affect its copyright protection in any way." (See, e.g., H. Rept. 94-1476 at 60)

**Comment by mbutterick** [comment]
_[I am not anyone’s lawyer and nothing I’m saying in this thread is legal advice. I’m just trying to straighten out these fonts for America.]_

> What cause of action is present and by whom if you're right about this OFL-1.1 issue?

I have no experience litigating software-licensing issues against the GSA. With that ignorance conceded, I’d suppose that:

1. If the GSA is violating the OFL for Libre Franklin, then the copyright holder of Libre Franklin could bring whatever claims arise from violation of that license (for instance, breach of contract, copyright infringement, could be others). 

2. Over time, as U.S. citizens create derivative works from Public Sans, the copyright holder of Libre Franklin could possibly have copyright-infringement claims against them as well (on the idea that if GSA violated the original OFL, then none of the derivative works were ever permitted). 

3. I could also imagine that these makers of derivative works could seek a declaratory judgment against the GSA that Public Sans is entirely in the public domain, and the copyright of Libre Franklin was implicitly terminated by inverse condemnation.

The main issue, however, is not the actual probability of this litigation, but rather the permanent risk that it _could_ occur. This is what puts a cloud over the legal status of Public Sans, which in turn affects everyone who will use the font. 

For instance, suppose that today, I make a derivative work from Public Sans. Can anyone from the GSA explain what rights I have in that work? What obligations? What kind of works can be derived from that new work? I truly have no idea, and no one in this thread has offered one, aside from conjectures that Public Sans lives in some handwavy twilight between OFL and public domain.

Sorry, but that’s no good. This is exactly the point the Open Source Initiative is making when it says “[public domain is not open source](https://opensource.org/node/878)”. Licenses create a chain of “certainty”; public-domain work cannot be licensed, so it breaks this chain.

> What rights would the government have under OFL-1.1 that they do not possess if public-sans is in the public domain?

I contend that the GSA can never comply with OFL-1.1: the OFL depends on assertion of copyright, and the GSA has no copyright in its contributions to Public Sans. So your hypothetical — the government having rights under OFL-1.1 — is impossible.

But let’s suppose I reframe your question from the perspective of someone who wants to make a derivative work:

* A font under OFL-1.1 creates an obligation on me to ensure my derivative work is “[distributed entirely under](https://opensource.org/licenses/OFL-1.1)” the OFL (and a cause of action by the copyright holder if I don’t). There are other conditions, but IMO that is the most significant.

* By contrast, if the font is in the public domain, then I have no obligations at all.

> How can this issue have any effect on the copyrights of related fonts it's arguably derived from when the stated legislative intent of Section 105 is that "publication or other use by the Government of a private work would not affect its copyright protection in any way." (See, e.g., H. Rept. 94-1476 at 60)

You omitted the sentence immediately before, [which says](http://uscode.house.gov/view.xhtml?req=(title:17%20section:105%20edition:prelim)): “there is nothing in section 105 that would relieve the Government of its obligation to secure permission in order to publish a copyrighted work”. If that permission has been secured, then yes, “publication or other use by the Government of a private work would not affect its copyright protection in any way.” I take this to mean that the US gov’t does not convert privately copyrighted material to public domain merely by handling it according to its permitted use — a safe-harbor protection against inadvertent exercise of eminent domain.

But that’s not the case here. The GSA began with Libre Franklin, a copyrighted work whose permissions are set out by OFL-1.1. The GSA has not complied (and cannot comply) with the obligations of OFL-1.1 (for reasons already explained). The safe harbor does not apply. This is why I think it’s possible that the GSA has inadvertently taken the copyright for Libre Franklin by eminent domain — this is the only legally coherent outcome.

When I opened this issue, however, I wasn’t thinking that the 17 USC § 105 issue affected the fonts upstream. Rather, it appeared that the GSA was attempting to assert copyright over public-domain work (by putting it under OFL-1.1), an act that would violate 17 USC § 105. It appears that’s probably not the case, though I don’t see that we’ve had an official answer from GSA about its position.

**Comment by wslack** [comment]
_again, IANAL, and speak only for myself_

> The GSA has not complied (and cannot comply) with the obligations of OFL-1.1 (for reasons already explained).

Ok, I think I understand now. You're saying that the OFL license asserts a requirement (that the modifications must receive the same license) that conflicts with US law for works by gov't employees that prevents that license. In other words, the US gov may not be able to label the transformations with the OFL license as required.

However, to be clear, the debate we're having is about **solely those transformations**. The copyright status of the original untransformed work is not in question, and the public domain nature of the transformations does not alter that. Its not legally coherent to take a question about those transformations and apply an answer outside the scope of inquiry. Work can be in the public domain and not usable because of how its entangled with copyrighted works. (Though, it might be usable solely as a description of a process that others can follow for modifying their own fonts).

Let me put it another way. If the US Gov't publishes a copyrighted photo without the proper licensing information, that work has not been magically spirited to the public domain because of a single person's mistake. (though it might create confusion, just as anyone else might if they republish a photo without the licensing information) Likewise, the US gov't can't accidentally spirit a font to the public domain just by transforming it a bit.

> So yes, I imagine US gov’t employees would need to step very gingerly around “transformations of copyrighted work”, lest they accidentally nudge those works into the public domain. 

The public domain work of transformation does not change the copyright status of the original work. That's (as far as I can see) a baseless assertion and I would welcome citations to the contrary.

**Comment by jackjennings** [comment]
> The public domain work of transformation does not change the copyright status of the original work. That's (as far as I can see) a baseless assertion and I would welcome citations to the contrary.

Imagine the alternative scenario where a derivative typeface has only modifications to the capital letters. If that typeface is now deemed public domain, the remaining unmodified glyphs have essentially been pulled into the public domain by virtue of the typeface as a whole being relicensed. (The classification of typefaces as software under US law means that the underlying data, not the overall design, is the basis of the copyright.)

Matthew, correct me if I’m misstating what you’re getting at.

**Comment by EssbieWGT** [comment]
Thank you for your thoughtful response! A couple additional questions: 

- Hasn't the gov't already "secure[d] permission in order to publish a copyrighted work" by relying on a work licensed under OFI-1.1? As OSI stated in the blog post you shared: "every open source license guarantees the freedom to innovate without seeking permission first." 
- Even if one argues GSA hasn't secured permission by virtue of reliance on an OSI approved license, how could one properly argue that Congress intended to combine the requirement that the government get permission with the idea that government use "would not affect its copyright protection in any way" when those two clauses seem by plain meaning to be separate, and indeed were separated in the section of the 1909 Copyright Act that comment in the legislative history clearly addresses? 
- Wasn't the copyright "transferred to GSA by assignment, bequest, or otherwise" when they relied on the OFI-1.1 license from the previous iteration of the font? 
- Are you aware of any cases where a federal agency has been found to have adopted a copyright via eminent domain? 
- Wouldn't your theory essentially mean that government can never properly secure an open-source license for any piece of software?

**Comment by mbutterick** [comment]
_[I am not anyone’s lawyer and nothing I’m saying in this thread is legal advice. I’m just trying to straighten out these fonts for America.]_

> Ok, I think I understand now. … the OFL license asserts a requirement (that the modifications must receive the same license) that conflicts with US law for works by gov't employees that prevents that license.

Yes, exactly.

> However, to be clear, the debate we're having is about solely those transformations. 

No, I’m afraid this premise is flawed. The OFL covering Libre Franklin does not allow the GSA (or any licensee) to conceptually isolate certain “transformations” and treat them specially. It only grants permission to make a “[Modified Version](https://opensource.org/licenses/OFL-1.1)”, and then sets certain conditions, e.g. that it be ”distributed entirely under this license”. Under the OFL, Public Sans must be a “Modified Version” of Libre Franklin, because there’s no other option. (Or, if you disagree, [what other possibility exists in the OFL](https://opensource.org/licenses/OFL-1.1)?) 

We cannot pretend that Public Sans is a discrete bucket of public-domain transformations sitting next to Libre Franklin. The OFL does not allow that characterization. (This argument is strained anyhow, since [you’ve already agreed](https://github.com/uswds/public-sans/issues/30#issuecomment-481520109) that the copyrighted and public-domain parts are an “inseparable mixture”.)

> The copyright status of the original untransformed work is not in question, and the public domain nature of the transformations does not alter that.

Because the GSA has not complied (and cannot comply) with the OFL, that seems like a premature conclusion.

> If the US Gov't publishes a copyrighted photo without the proper licensing information, that work has not been magically spirited to the public domain because of a single person's mistake

This is a misleading analogy. First, the GSA has not made a mistake. Second, the GSA is inviting U.S. citizens to make derivative works from Public Sans based on a misrepresentation (namely, that it is [licensed under the OFL](https://github.com/uswds/public-sans#license), which it can never be). Again, I invite anyone to explain: if I make a derivative work from Public Sans, under what terms can I distribute this new work? (Don’t say “the same” 🙄)

---

> Hasn't the gov't already "secure[d] permission in order to publish a copyrighted work" by relying on a work licensed under OFI-1.1? 

No. This permission is contingent on meeting the conditions that the OFL places on derivative works. Which the GSA has not done (and cannot do).

> how could one properly argue that Congress intended to combine the requirement that the government get permission with the idea that government use "would not affect its copyright protection in any way" when those two clauses seem by plain meaning to be separate, and indeed were separated in the section of the 1909 Copyright Act that comment in the legislative history clearly addresses?

I will leave that question to better lawyers. [Here’s one IP lawyer](http://www.ipprospective.com/copyright-caucus/intellectual-property-and-eminent-domain-a-plausible-combination/) who sees no reason why a copyright couldn’t be taken by eminent domain.

> Wasn't the copyright "transferred to GSA by assignment, bequest, or otherwise" when they relied on the OFI-1.1 license from the previous iteration of the font?

No. “Transferred” connotes a change in ownership of the copyright. The OFL merely confers certain rights.

> Are you aware of any cases where a federal agency has been found to have adopted a copyright via eminent domain?

I had never thought about the issue until two days ago. Off the top of my head, I’m only aware of the compulsory patent-licensing provisions of [28 USC § 1498](https://www.law.cornell.edu/uscode/text/28/1498) — not the same, but an example of where federal agencies reconfigure private IP rights. Also, compulsory mechanical licensing for musical compositions under [17 USC § 115](https://www.law.cornell.edu/uscode/text/17/115) is arguably a light form of eminent domain in copyrights, inasmuch as it deprives the licensor of the right to decline a license.

> Wouldn't your theory essentially mean that government can never properly secure an open-source license for any piece of software?

No. My argument is much smaller: that in this case, the GSA has not (and cannot) comply with a particular license (the OFL), because it needs the GSA to assert a copyright in its contributions, which the GSA cannot do. 

We might imagine a number of ways the GSA could’ve avoided this pickle — for instance, by studying how the Smithsonian handled the development of the [Cooper Hewitt typeface](https://www.cooperhewitt.org/open-source-at-cooper-hewitt/cooper-hewitt-the-typeface-by-chester-jenkins/). But, having put itself in the pickle, the GSA now needs to find a way out. Otherwise, a legal cloud will continue to hang over the fonts.

**Comment by mbutterick** [comment]
_[I am not anyone’s lawyer and nothing I’m saying in this thread is legal advice. I’m just trying to straighten out these fonts for America.]_

> Even if one argues GSA hasn't secured permission by virtue of reliance on an OSI approved license, how could one properly argue that Congress intended to combine the requirement that the government get permission with the idea that government use "would not affect its copyright protection in any way" when those two clauses seem by plain meaning to be separate, and indeed were separated in the section of the 1909 Copyright Act that comment in the legislative history clearly addresses?

PS, with the caveat that I may still be misunderstanding your question, and I am not a better lawyer than yesterday. The whole sentence from the [committee notes](http://uscode.house.gov/view.xhtml?req=(title:17%20section:105%20edition:prelim)) reads as follows:

> The committee here observes: (1) there is nothing in section 105 that would relieve the Government of its obligation to secure permission in order to publish a copyrighted work; and (2) publication or other use by the Government of a private work would not affect its copyright protection in any way.

Maybe you’re suggesting that the US gov’t still benefits from a safe harbor against inadvertent exercise of public domain. OK, let’s suppose that it does. How does that fix the problem here? The GSA is still failing to comply with the OFL. Eminent domain is arguably a way to cut the Gordian knot (regardless of how it is triggered).

Or maybe you’re suggesting that the US gov’t cannot take copyrights by eminent domain at all, because these committee notes say that any government “use … would not affect its copyright protection in any way”. The requirement that the US gov’t pay “just compensation” after taking private property for public use comes from the US Constitution, of course. Therefore, if the “use” amounts to a taking under Amendment V, these committee notes can’t insulate the US gov’t from its constitutional obligation. That would be an unconstitutional interpretation.

**Comment by mbutterick** [comment]
_[I am not anyone’s lawyer and nothing I’m saying in this thread is legal advice. I’m just trying to help sort out these fonts for America.]_

BTW2, because there was a reference to [DoD guidelines](https://dodcio.defense.gov/open-source-software-faq), I had a closer look. 

Yes, [the DoD says](https://dodcio.defense.gov/open-source-software-faq/#Q:_Can_government_employees_contribute_code_to_open_source_software_projects.3F) that government employees can contribute to open-source software projects and that “the resulting work is a ‘joint work’’ under [17 USC § 101](https://www.law.cornell.edu/uscode/text/17/101). However, this guideline comes with the proviso that the work be released “according to the terms of the original open-source license”. Not all open-source licenses are the same, so further scrutiny is needed.

Let’s suppose the license is the GNU GPL. Though the Free Software Foundation does consider copyright a [prerequisite](https://www.gnu.org/licenses/copyleft.en.html) for licensing, it also considers [public domain work to be compatible](https://www.gnu.org/licenses/license-list.en.html#PublicDomain) with the GNU GPL. Moreover, though it doesn’t seem to be part of the [GPL per se](https://www.gnu.org/licenses/gpl-3.0.en.html), the FSF says that it’s “[no problem](https://www.gnu.org/licenses/gpl-faq.en.html#GPLUSGovAdd)” for US government employees to contribute to GPL-licensed projects. 

Fair enough — if Public Sans had been based on a GPL-licensed font, or font under a GPL-compatible license, then there would be no issue.

But it’s not. According to the FSF, the OFL is [not compatible with the GPL](https://github.com/FortAwesome/Font-Awesome/issues/1124#issuecomment-29507471). 

The [OFL is approved](https://opensource.org/licenses/OFL-1.1) by the Open Source Initiative. Compared to the FSF, the Open Source Initiative takes a [generally more cautious view](https://opensource.org/faq#public-domain) about public domain. Unlike the FSF, the OSI [does not endorse](https://opensource.org/node/878) the combination of open-source licensed software and public-domain software. Also unlike the FSF, the OSI has not endorsed special status for US government contributions.

So I can understand the reasoning behind the GPL-style argument — namely, that government employees [can make contributions](https://www.gnu.org/licenses/gpl-faq.en.html#GPLUSGovAdd) that don’t affect the license of the original work. That is indeed the policy that the FSF applies to the GPL. 

But since the OFL is not GPL-compatible, and doesn’t live under the FSF umbrella, the outcome is not automatically the same. 

This is also why I described my argument here as “small”: it is particular to the OFL, which I imagine is not a license the US gov’t has directly used much (or maybe ever).

**Comment by davelab6** [comment]
I am not a lawyer and this is my personal opinion, not a reflection on my employer (Google); but I am a recognized expert on libre font licensing.

@mbutterick Thanks for filing these issues! I've loved reading your comments on this issue tracker and on your email mailing list about this project, and I think it's all an excellent continuation of your unrelenting critique of libre font movement :) However, just like always, I think you are completely wrong 😂

@tssva wrote:

> The OFL-1.1 only allows distribution of derivative works if the derivative is itself licensed under the OFL-1.1, so if you take the view that public-sans is a combination of public domain licensed and OFL-1.1 licensed work than distribution of it is a violation of the original works OFL-1.1 license.

"public domain licensed" is a contradiction in terms. 

Works in the public domain are not licensed for use or distribution, they are used and distributed directly because there is no artificial restriction on doing so, as there would be of the work was subject to copyright restrictions. 

Therefore I believe the licensing of Public Sans is not in doubt, because where the work has copyrights adhered, they are licensed under the OFL and only the OFL, and where the work has no copyright adhered, they are not licensed, but that doesn't conflict with the licensed parts - because the public domain is not a license.

**Comment by tssva** [comment]
@davelab6 wrote:

> Therefore I believe the licensing of Public Sans is not in doubt, because where the work has copyrights adhered, they are licensed under the OFL and only the OFL, and where the work has no copyright adhered, they are not licensed, but that doesn't conflict with the licensed parts - because the public domain is not a license.

The OFL-1.1 stipulates:
> 5) **The Font Software, modified or unmodified, in part or in whole,
must be distributed entirely under this license**, and must not be
distributed under any other license. The requirement for fonts to
remain under this license does not apply to any document created
using the Font Software.

How can what is clearly a modified work be distributed as partially under the OFL-1.1 license and partially unlicensed still be in compliance with the OFL-1.1 requirement that any modified works be distributed entirely under the OFL-1.1 license?

**Comment by otherjoel** [comment]
Within the US, the Public Sans font is public domain, as a matter of law. This much is extremely clear, and the maintainers should change the docs to acknowledge this.

I find the question of how this affects the original font curious and fascinating. But I can’t see that it’s necessary or even possible for any of this project’s maintainers to settle that issue. Isn't it a case of “what's done is done”?

**Comment by davelab6** [comment]
> How can what is clearly a modified work be distributed as partially under the OFL-1.1 license and partially unlicensed still be in compliance with the OFL-1.1 requirement that any modified works be distributed entirely under the OFL-1.1 license?

"Partially unlicensed" normally means "all rights reserved" which is the default when copyrights have adhered to the work. But here all work subject to copyright is licensed wholly under the OFL, and not any other license. 

Problem solved!

**Comment by davelab6** [comment]
> Within the US, the Public Sans font is public domain, as a matter of law

Kindly, this is bonkers :) The work authored by the usg are public domain, but since that work modifies the underlying work to which the Libre Franklin authors own copyrights that are licensed under OFL, the combined whole is definitely not public domain.

**Comment by tssva** [comment]
> > How can what is clearly a modified work be distributed as partially under the OFL-1.1 license and partially unlicensed still be in compliance with the OFL-1.1 requirement that any modified works be distributed entirely under the OFL-1.1 license?
> 
> "Partially unlicensed" normally means "all rights reserved" which is the default when copyrights have adhered to the work. But here all work subject to copyright is licensed wholly under the OFL, and not any other license.
> 
> Problem solved!

No problem solved. You haven't at all addressed the core issue that the license of the work subject to copyright only grants permission for distribution if the complete modified work is distributed under the OFL-1.1 which is clearly not the case here.

**Comment by davelab6** [comment]
Initially you are explicit that the license extends only to "the work **subject to copyright**", but then you refer to "the complete modified work". 

Why don't you say "the complete modified work subject to copyright"?

**Comment by otherjoel** [comment]
> Why don't you say "the complete modified work subject to copyright"?

The OFL neither makes nor permits such a distinction, this has [already been addressed above](https://github.com/uswds/public-sans/issues/30#issuecomment-482000759).

**Comment by davelab6** [comment]
The OFL doesn't have to: the distinction is inherent in the law. Public
domain is not a license.

**Comment by mbutterick** [comment]
_[I’m not anyone’s lawyer and nothing I’m saying in this thread is legal advice. I’m just trying to help sort out these fonts for America.]_

> The OFL doesn't have to: the distinction is inherent in the law. Public domain is not a license.

Respectfully @davelab6, yours is becoming a truth-by-repetition argument. You keep jousting with words of other commenters rather than grappling with the plain language of the OFL already cited:

> 5. The Font Software, modified or unmodified, in part or in whole,
must be distributed entirely under this license, and must not be
distributed under any other license.

Here’s the question you’ve yet to answer, and since you hold yourself out as a “recognized expert on libre font licensing”, I’m sure we’re all interested to hear:

What supports your theory that 1) public-domain modifications are exempt from the OFL’s explicit, unqualified requirement that the modified font “must be distributed entirely under this license”, and that 2) this relicensing requirement only pertains to (in your words) “work subject to copyright”?

Yes, of course you’re right that “public domain is not a license”. But that is merely the predicate for the question above, not the answer.

**Comment by malanthomasii** [comment]
_[IANAL, but I'm a librarian who has done some copyright work such as writing issue briefings for publication by a committee of the American Library Association.]_

I find myself mostly agreeing with @mbutterick here. Within the U.S., I think that the following is true:

1. There is some case law descended from ProCD, Inc. v. Zeidenberg suggesting that a public domain work could be locked behind a license (contract) where agreement to be bound by that license (contract) is a prerequisite for gaining access to a copy. I don't _like_ it, but it's there.
2. I'm not sure that a line of clickwrap license cases are applicable to GitHub, however, as I don't recall personally agreeing to abide by the license terms on this project as a condition of accessing the code, nor do I see a requirement to do so in GitHub's TOS, so a copy could be exported without agreeing to the license.
3. Given that, absent a clickwrap agreement, part or all of the derivative work is effectively not being distributed entirely under the license, this would violate the terms of the license on the original work.
4. The derivative work thus represents a copyright violation.
5. Whether the derivative work is in the public domain is an interesting question, and I'm not certain what the answer is. I lean towards "probably."
6. However, I don't think that the original work is placed into the public domain by this chain of events. That is, to my mind, a non-issue, but I wanted to state it for clarity.

**N.b.:** The license also contains a copyright statement, but it only names what I assume is the original work's rightsholder and not the derivative work's (if any). I'm not sure what that's meant to imply, although it might be a violation of 17 U.S. Code § 506(c). Have fun with that one!

**Comment by impallari** [comment]
Hi all, Pablo from LibreFranklin here.
I'm happy to help in any way needed, just let me know. I'm open to suggestions to make this work

**Comment by clacke** [comment]
Could a third party take the modifications in isolation from the public domain and license them under the OFL?

**Comment by medicalwei** [comment]
I think one way to resolve this issue is to request @inpallari to redistribute the font under his own copyright, which he can claim he incorporated all the changes that US government made contributions to LibreFranklin.

IANAL

**Comment by malanthomasii** [comment]
@impallari: The intervention of the original work's rightholder (Impallari Type) could solve at least one problem. The OFL is not the only terms on which the original work can be distributed by its original rightsholder; it is merely the only option offered to the general public thus far. If the government receives separate permission—however belatedly—to use the work as they already have, then the government has not breached the OFL (because they are not acting under it) or infringed upon copyright in the U.S.

The only remaining question, then, would be whether the derivative work is completely public domain in the U.S. I continue to go back and forth on that in my head, but I think that the following is true if such permission is given:

1. The derivative work must continue to be offered under the OFL for non-U.S. users, as this entire issue is U.S.-specific. Therefore, it is necessary to keep some sort of reference to the OFL.
2. If the derivative work is not completely public-domain in the U.S., then the remaining parts must continue to be offered under the OFL as well.
3. If the derivative work is completely public-domain in the U.S., then U.S. users would be free to ignore the OFL and use the derivative work. However, for reasons stated both earlier in this comment and in my prior comment, this doesn't require removing the references to the OFL.
4. Until we find someone or some line of cases to clarify the application of §&nbsp;105 to derivative works, the legally safe thing for subsequent users to do would be to assume that the OFL is necessary and comply with it.

Thus, I feel that this issue would be adequately addressed by a rightsholder of the original work granting permission to the U.S. Government to do what it has already done.

While I do have an academic interest in the result of the inquiry into the status of derivative works under §&nbsp;105, I don't think that it would present an obstacle to proceeding once such permission is given, because the OFL will be necessary for some users regardless of the result of that inquiry and thus should remain. A note that the project "might or might not be partially or wholly within the public domain in the United States" would be nice and completely sufficient for any lingering doubts in my mind.

**Comment by xorhash** [comment]
As an aside: I'd just like to chime in and point out that you may still want *some* form of license as a fallback.

In other jurisdictions, copyright  may still apply to Public Sans in particular; the United States of America may be recognized as a copyright holder of Public Sans. If Public Sans is meant to be usable by entities outside the U.S., it may be worthwhile to have the OFL specified for jurisdiction where it is subject to copyright.

(Disclaimer: I am not a lawyer. This is not legal advice.)

**Comment by mbutterick** [comment]
_[I am not anyone’s lawyer and nothing I’m saying in this thread is legal advice.]_

> A note that the project "might or might not be partially or wholly within the public domain in the United States" would be nice and completely sufficient for any lingering doubts in my mind.

Respectfully @malanthomasii, from a software-licensing perspective that would be the worst possible outcome. The whole premise of this issue is that the GSA has put Public Sans into an uncertain legal state. Papering over the problem with some wibbly-wobbly language about what “might or might not” be true will only multiply the doubts. This is why, as the OSI puts it, “[certainty](https://opensource.org/node/878)” is an indispensable ingredient of software freedom.

**Comment by malanthomasii** [comment]
@mbutterick: That statement was intended to be conditional on my other conclusions being true and the intervention of the rightsholder to heal the government's intervening copyright violation, but to address your point more directly:

Yes, the situation is uncertain. Many copyright situations are. It would not be wise or ethical to pretend otherwise. Until the uncertainty regarding §&nbsp;105 and derivative works is clarified, pretending that something that might be public domain is definitely copyrighted is highly objectionable to me as a matter of clarity and as a matter of ethics with regards to the public domain. I would therefore like a disclaimer for so long as the situation remains uncertain.

Your link points out the benefits of having a fallback license where the public domain may be unavailable. I agree with that point. I am not calling for a waffling disclaimer as the sole solution to the problem; I think that if the government's violation can be healed so that the only remaining question is on what terms further use may be made, a license should remain a option for all of the reasons that I have previously stated.

As a practical matter, operating under a "worst-case" assumption is a regular practice in other areas of copyright uncertainty, and for someone who's uncertain to choose to abide by a license that might or might not be legally necessary or enforceable in all instances isn't impossible or even that odd to me. I think that we can render academic the uncertainty in this (highly fact-specific) case, allowing for practical use while the uncertainty remains unresolved.

**Comment by mbutterick** [comment]
_[I am not anyone’s lawyer and nothing I’m saying in this thread is legal advice.]_

> operating under a "worst-case" assumption is a regular practice in other areas of copyright uncertainty, and for someone who's uncertain to choose to abide by a license that might or might not be legally necessary or enforceable in all instances isn't impossible or even that odd to me.

I see your point, though the worst case is quite a bit worse than you suggest. We’re not in a situation where “at best” the fonts are public domain, and “at worst” they’re under the OFL. If the Libre Franklin OFL was violated, then the Public Sans fonts are a unpermitted derivative work, and amount to one big copyright violation. 

Still, if you think there’s some language that would “allow[] for practical use”, I encourage you to [open a separate issue](https://github.com/uswds/public-sans/issues/new), since it concerns the current state of the fonts, whereas this issue concerns their future state.

**Comment by pbhj** [comment]
>     3. If the derivative work is completely public-domain in the U.S., then U.S. users would be free to ignore the OFL and use the derivative work. However, for reasons stated both earlier in this comment and in my prior comment, this doesn't require removing the references to the OFL.

Surely though if a work is public domain then there are no copyright restrictions, in USA at least, and a USA based person can [re-]publish the work CC0, or public domain, and thus make it available globally from that source regardless of whether USA government retain some copyright on their version in other countries, it doesn't matter as the people aren't copying their typeface but are copying John Does' published work? With copyright it matters if you copy, if you don't copy from the USA owned work in some other jurisdiction but copy from the CC0, say, work of John Doe (which is really a re-publish of a "public domain" work) then there's no case to answer against USA government.

I realise it's a slight side-issue but it would circumvent some of the shenanigans (publishing a "public domain [but only in USA]" work as also OFL).

**Comment by malanthomasii** [comment]
@mbutterick - You can't separate the pre-existing copyright violation from the problem of whether people can make use of this distribution in the future; they both derive from the intersection of §&nbsp;105 and the OFL, and the underlying violation in creating and publishing the derivative work is sufficient to create legal uncertainties in any "future state" of the fonts. I have suggested that @impallari can heal the underlying violation. If that is not how the law works, or you have a better fix in mind, I am open to being educated.

@pbhj - That will not work. If a work is out of copyright in Country A, and you make a copy of it in Country A, it is still the same work and not a new one. If that work is copyrighted in Country B, it is still copyrighted in Country B by the original copyright holder. You cannot distribute that work to people in Country B because you do not have that (copy)right; someone else has that (copy)right. You cannot give up the copyright in Country B because you don't own the copyright in Country B. You cannot hijack someone else's copyright in Country B by republishing the work in Country A. Someone in Country B generally does not have the right to make a copy of the work or any other use of it in Country B regardless of where they get their copy. (There are some slight exceptions that are irrelevant here.)

**Comment by david-a-wheeler** [comment]
Whew, a lot of discussion.  Let me first scope this down: Outside the US, the US can assert copyright and release its changes under the OFL-1.1 like anyone else.  The US appears to have done so, since it says that the whole thing is under the OFL-1.1.  The complication discussed here *only* applies within the US, where the changes by US government employees as such *cannot* be copyrighted by law.  So let's focus on that.

The complication is due to the OFL-1.1's license text, which is very different from licenses that are used by most programs (MIT, GPL, etc.).  It says:

> 5) The Font Software, modified or unmodified, in part or in whole, must be distributed entirely under this license, and must not be distributed under any other license

Again, this text is totally different from commonly-used licenses.  OFL-1.1 may be common for fonts, but in the wider world of software it's not a common license..  So it should be unsurprising that there are some oddities that create complications when examining an unusual license.

Personally I think this text is poorly drafted, because one obvious interpretation isn't actually possible.  Namely, you can't say that every part that you distribute is copyrighted.  For example, if I extracted one character from the software, that character would be a part, and yet any one character can't be copyrighted (it's in the public domain) & thus its distribution doesn't need a license at all.  If parts were developed at different times, and they fall out of copyright at different times, this could be read as making distribution illegal after a while, and that doesn't seem to be the intent at all.  So it seems to me that the "parts" considered here don't seem to have anything to do with what's being contributed.  Instead, it has to do with what happens if someone extracts any portion of it that happens to also be covered by a license.  In short, if you receive something that as a whole is licensed under the OFL-1.1, and it has a license at all, it must be licensed under the OFL-1.1.  Saying "it must be under a license" is essentially inoperative: if you extract a part that isn't covered by copyright, there is no license that can be enforced to prevent it.

So now let's look at the phrase "distributed entirely under this license" and see if we can find a more reasonable interpretation of the clause, since the interpretation we just considered doesn't make sense.  I think a better and more reasonable interpretation is that text means that the work as distributed is *entirely* under the OFL-1.1 license.  That is, you can't add new clauses to the license to reduce the permissions granted to the recipient.  It says "modified or unmodified" but that's straightforward - even if the resulting work is modified, you still get the rights & responsibilities of the OFL-1.1 (no more, no less). The oddity is when you see "in whole or in part", but I think there's a simple interpretation: if it's licensed at all, and you distribute a part, then that thing you distributed has to be under the OFL-1.1.  But what if the part you extract is in the public domain?  That's easy: If some part is in the public domain, and only that part is distributed, then no license (including the OFL-1.1) can change anything: that part is in the public domain.  The license cannot prevent you from extracting parts that are not copyrighted, because once there's no copyright, there's no legal mechanism in copyright to prevent its redistribution.

I think the solution to understanding this license is that "distribution" is different from "modification".  The US government additions do not have a copyright in the US, so those modified portions don't have a copyright in the US.  But the entire modified work is under the OFL-1.1, because it includes parts that are under the OFL-1.1.  That's no different than how it started: the original work (before US government modifications) included parts that were not under copyright, because you can't copyright single characters.  Nothing in the license text forbids adding parts of uncopyrighted material, nor can it, since all copyrighted materials are built from combinations of uncopyrightable material.  The license controls *distribution* of the *result*.  The work as a whole (when combined) is still under copyright, so the OFL-1.1 is satisfied.  When a part is extracted, it's either copyrighted (so the OFL-1.1 is satisfied) or it's not (in which case no copyright license can re-copyright it, and thus the OFL-1.1 cannot prevent it).  The OFL-1.1 cannot imply that all parts of a work be copyrighted, because if you break anything into small enough parts then every single one of them is not subject to copyright.

This whole thing is very fact-specific to OFL-1.1.  The widely-used OSS licenses don't have these odd text clauses that require puzzling through when you're combining public domain with copyrighted material.  

Caveat: I'm not a lawyer, I'm certainly not a government lawyer, and I don't have a lot of time to devote to this discussion sadly.  These are just my personal opinions, and I reserve the right to change my mind if someone provides me a good argument :-).  That said, I think this resolves the issue.

**Comment by tssva** [comment]
@david-a-wheeler I don't know where you get the idea that you can't copyright a single character. That single character contains computer code as far as copyright law is concerned and is certainly still under copyright. Even the characters modified by government still contain code from the original font and therefore aren't completely public domain when extracted individually.

**Comment by malanthomasii** [comment]
@tssva I'm going to assume for the sake of argument that, because GitHub is a code repository and he's talking about other software licenses by comparison, that he means a single character of code. Take this example:

`&nbsp;`

He means the ampersand character in the code, not the non-breaking space that results.

Unless he doesn't, in which case you're right.

**Comment by david-a-wheeler** [comment]
@malanthomasii - correct. What I mean is that you cannot copyright a single character in complete isolation.

**Comment by mbutterick** [comment]
_[I’m not anyone’s lawyer and nothing I’m saying in this thread is legal advice.]_

> if you break anything into small enough parts then every single one of them is not subject to copyright … I think this resolves the issue

It “resolves the issue” only by detonating the very idea of copyright in literary works (a category that includes software code). By your argument, if we just zoom in far enough on any unauthorized copying of a literary work, we can claim that we’re merely copying “[single character](https://github.com/uswds/public-sans/issues/30#issuecomment-492195038)[s]” that are individually “not subject to copyright”. _Poof_ — no copyright violation is ever possible.

(Leaving aside that this theory seems to contradict the factual representations the GSA [has already made](https://github.com/uswds/public-sans/blob/master/README.md) about the relationship of Public Sans and Libre Franklin.)

**Comment by david-a-wheeler** [comment]
> By your argument, if we just zoom in far enough on any unauthorized copying of a literary work, we can claim that we’re merely copying “single character[s]” that are individually “not subject to copyright”.

Not at all.  A single character in isolation is not subject to copyright, because any one character in isolation cannot be original.  Combinations of characters of course potentially *are* subject to copyright.  My point is that claiming that any possible part *must* be subject to copyright is manifestly absurd, because there is a single character that is "part" of a larger work, and can be extracted out of the larger work, yet a single character in isolation is not subject to copyright.  Therefore, interpreting the license clause that "all possible parts are subject to this license" is absurd.

A better interpretation is that the "entirely" in the clause "must be distributed entirely under this license, and must not be distributed under any other license" means "this work (modified or not) must be under this license and not a different license".  That seems far more logical, since "the entirety of every possible part no matter how small must be under this license" is not actually possible. Under this reading of "entirely" there is no problem.  No license at all is not a different license - it's not a license at all.  Therefore, there's no conflict.

**Comment by mbutterick** [comment]
_[I’m not anyone’s lawyer and nothing I’m saying in this thread is legal advice.]_

> interpreting the license clause that "all possible parts are subject to this license" is absurd.

i don’t recall anyone making this argument. The principal question in this thread pertains to the OFL requirement that the modified work “must be distributed entirely under this license”, which is incompatible with the fact that the GSA’s modifications are public domain, and therefore cannot be placed under any license.

> A better interpretation is that the "entirely" in the clause "must be distributed entirely under this license, and must not be distributed under any other license" means "this work (modified or not) must be under this license and not a different license".

I don’t recall anyone seeking to debate the meaning of “entirely”.

> No license at all is not a different license - it's not a license at all. Therefore, there's no conflict.

Correct, there is no apparent conflict between public-domain modifications and the part of the OFL that requires that the modifications “must not be distributed under any other license”. But again, that’s not the troublesome phrase, as mentioned above [and elsewhere](https://github.com/uswds/public-sans/issues/30#issuecomment-485272741).

In any case, if the GSA agrees with your argument, I’m sure we’d all be interested to hear that from them, since they’re the only ones who can rectify the problem.

**Comment by tssva** [comment]
> In any case, if the GSA agrees with your argument, I’m sure we’d all be interested to hear that from them, since they’re the only ones who can rectify the problem.

@mbutterick - Given the lack of any response I assume the plan of the USWDS at this point is to ignore the issue and hope it goes away. Any real movement would probably require reporting the issue to the GSA OIG.

**Comment by david-a-wheeler** [comment]
@mbutterick:

> The principal question in this thread pertains to the OFL requirement that the modified work “must be distributed entirely under this license”...

My current thinking is that it *is*, because no other license is being used.  What's more, the work in its entirety *is* being distributed under exactly that license without change to the license (which I think is the intent of the "no other licenses" part). Done and done.

In addition, I suspect a court will try to move forward by trying to figure out intent of the license (not just its raw text) when trying to interpret it.  The intent of the license is stated in the preamble: "The goals of the Open Font License (OFL) are to stimulate worldwide development of collaborative font projects, to support the font creation efforts of academic and linguistic communities, and to provide a free and open framework in which fonts may be shared and improved in partnership with others."  It also says that the " The fonts and derivatives, however, cannot be released under any other type of license."... and since the work as a whole continues to be under the same license, that also is met.

So I think there's an interpretation of the license text that has been met (and I think it's quite plausible), and the overall goal of the license is being met.  That doesn't sound like a problem.

> In any case, if the GSA agrees with your argument, I’m sure we’d all be interested to hear that from them, since they’re the only ones who can rectify the problem.

Clearly people disagree on whether or not there is a problem, and the issues are extremely specific to the OFL-1.1.  The question would not even come up with more widely-used licenses such as MIT, BSD-2-Clause, BSD-3-Clause, Apache-2.0, LGPL, or GPL, because none of them have this unusual (quirky!) clause.

It'd be awesome if USWDS released a formal opinion of counsel (or similar) that explains why they believe it's fine.  I do not have the power to force that :-).

**Comment by thisisdano** [comment]
Addressed in https://github.com/uswds/public-sans/pull/123
